developers/wcs/embeddings/models.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Choose a model
+sidebar_label: Choose a model
+sidebar_position: 2
+image: og/wcs/user_guides.jpg
+---
+
+On this page, you can find a list of pre-trained models designed specifically for enterprise retrieval tasks in English and other languages. Additional models and features will be added in the future, so please check back regularly for updates.
```

Comment:
I think the page would benefit from explicitly saying that it contains *the* list of available models for Weaviate Embeddings.